tests/CMakeLists.txt

```diff
@@ -49,4 +50,34 @@ target_link_libraries(${TARGET_NAME}
         gtest_main
         date::date
         date::tz
+)
+
+# target [dots-unittests-experimental]
+add_executable(${TARGET_NAME_EXPERIMENTAL} src/TestStaticStructExperimental.cpp)
```

Comment:
Ich denke mal dieses neue Test-Target soll auch ausgeführt werden. Weißt du wie du es für gitlab und github einfügst?

Reply:
Anscheinend läuft das ja über die `build.yml`. Kann man mehrere `run`-Einträge verwenden oder muss ich einen neuen Test-Block erstellen?

Reply:
https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsrun
Ich lese das so, dass man pro Step einmal "run" aufrufen kann. Also müsstest du einen weiteren Step anlegen.